bin/puppet_webhook

```diff
@@ -6,7 +6,15 @@ require 'webrick'
 require 'webrick/https'
 require 'puppet_webhook'
 
-config_file '../config.yml'
+server_conf = File.join(__dir__, '..', 'config', 'server.yml')
+
+if File.exist?('/etc/puppet-webhook/server.yml')
```

Comment:
The only problem with a conditional like this is it's all or nothing. To override a single setting, you have to override all. My preferred solution is to load the defaults first, then load & merge the custom config if it exists.

Reply:
I agree with you on that. I'll have to look into and see if Sinatra::Config_File allows for merging of config from multiple files. Not sure it does.